CHANGELOG.md

```diff
@@ -6,14 +6,16 @@
 
 ### Bug Fixes
 
+* [ObjectServer] Fixed a crash when an authentication error happend (#4726).
```

Comment:
This isn't entirely correct. I would clarify this so "Fixed a crash when debug level logging was enabled and an authentication error happend"